README.md

```diff
@@ -1,41 +1,26 @@
-# OpenAI Search
+# Chatbot Öffentliches Gestalten
 
-This is a Proof Of Concept (POC) for a search engine that uses OpenAI's GPT- (3-4) API to generate search results for the project Handbuch öffentliches Gestalten. Based on https://github.com/supabase-community/nextjs-openai-doc-search and https://levelup.gitconnected.com/how-to-stream-real-time-openai-api-responses-next-js-13-2-gpt-3-5-turbo-and-edge-functions-378fea4dadb (https://archive.is/2ouIz)
+This is a Proof-Of-Concept (POC) for a webapp using OpenAI's GPT (3-4) API as chatbot to answer questions about the book "Handbuch öffentliches Gestalten" (available online here: https://www.oeffentliches-gestalten.de/).
 
-## Getting Started
+This POC was based on https://github.com/supabase-community/nextjs-openai-doc-search.
+
+It is available here: https://chat.oeffentliches-gestalten.de/
 
-First copy the files from [docs](https://github.com/technologiestiftung/oeffentliches-gestalten-docusaurus/tree/main/docs) folder of the GitHub repo [technologiestiftung/oeffentliches-gestalten-docusaurus](https://github.com/technologiestiftung/oeffentliches-gestalten-docusaurus) to the pages directory.
+## Getting Started
 
 Start the supabase project.
 
 ```bash
 supabase start
 ```
 
-Copy the `.env.example` file to `.env` and add the variables and api keys.
+Copy the `.env.example` file to `.env` and add the missing variables and api keys.
 
 Then run the development server:
 
 ```bash
 npm ci
 npm run dev
-
-```
-
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-Regenerate the embeddings.
-
-```bash
-npm run embeddings
```

Comment:
I think the part about generiting the embeddings should stay. Since without it there will be no output

Reply:
okay, will fix it  👍